src/components/containers/seo.js

```diff
@@ -173,7 +172,7 @@ const SEO = ({ description, meta, title, no_index, has_organization_schema }) =>
                         clientToken: 'pubc42fda54523c5fb23c564e3d8bceae88',
```

Comment:
please add these token to utility, and imo its better to put it in gatsby browser rather than in seo, wdyt?